ui/de-lib/src/main/java/org/iplantc/de/diskResource/client/views/metadata/DiskResourceMetadataViewImpl.java

```diff
@@ -329,6 +383,19 @@ public String getKey(Avu item) {
 
     }
 
+    private void setButtonState() {
+        if (selectedSet.size() == 0) {
+            deleteMetadataButton.disable();
+            editMetadataButton.disable();
+        } else if (selectedSet.size() == 1) {
+            deleteMetadataButton.enable();
+            editMetadataButton.enable();
+        } else {
+            deleteMetadataButton.enable();
+            editMetadataButton.disable();
+        }
+    }
```

Comment:
These buttons probably should be enabled or disabled depending on the `writable` boolean too.
So I would create 2 local booleans here, and call `deleteMetadataButton.setEnabled(deleteEnabled)` and `editMetadataButton.setEnabled(editEnabled)`, like we do elsewhere.